Source/Shaders/Builtin/Structs/materialInput.glsl

```diff
@@ -25,4 +25,5 @@ struct czm_materialInput
     float height;
     float slope;
     float aspect;
+    vec4 layerColor[czm_layerColorNumber];
```

Comment:
Should there be a `layerColorNumber.glsl` file? I had to create one locally to test this. Also a sandcastle demo would be helpful. I modified the Globe Materials sandcastle.

Reply:
Sorry, I forgot to check in layerColorNumber.glsl
I would like to find the best way to implement this and make sure it just makes sense at all before writing any demo. This is why your input is very valuable here.